variables.tf

```diff
@@ -142,12 +148,6 @@ variable "allow_ssl_requests_only" {
   description = "Set to `true` to require requests to use Secure Socket Layer (HTTPS/SSL). This will explicitly deny access to HTTP requests"
 }
 
-variable "versioning_mfa_delete_enabled" {
-  type        = string
-  default     = false
-  description = "Enable MFA delete for the bucket"
-}
```

Comment:
Why do we remove mfa ?

Reply:
Oh because the s3 bucket doesn't support it... Can we keep this input variable at least as a deprecated input ?

Reply:
See the PR description, second bullet point.